*** ./smali/com/android/server/LightsService.smali	2016-01-12 21:06:01.276822770 +0800
--- /home/ghoul/micode/sence/temp/new_smali/services.jar.out/./smali/com/android/server/LightsService.smali	2016-01-12 21:05:00.467602278 +0800
***************
*** 207,209 ****
--- 207,222 ----
  
      return-object v0
  .end method
+ 
+ .method public setLight(ILcom/android/server/LightsService$Light;)V
+     .locals 1
+     .param p1, "index"    # I
+     .param p2, "light"    # Lcom/android/server/LightsService$Light;
+ 
+     .prologue
+     iget-object v0, p0, Lcom/android/server/LightsService;->mLights:[Lcom/android/server/LightsService$Light;
+ 
+     aput-object p2, v0, p1
+ 
+     return-void
+ .end method
